Log outlet filters (#657)
circonus-labs/libmtev: docs-md/config/logging.md

is setup to output to "stderr".  Also, because we did not declare a "debug"
log_stream with `disabled="false"`, the default state remains disabled.

> Outlets may have a `filter` attribute that will apply filters to messages passing

Reviewer: ```suggestion
Outlets may have a `filter` attribute that will apply to messages passing
```

along these graph edges.  A filter is a logical expression using `mtev_logic` grammar.

> It suppose `and(p[,p[,...]])`, `or(p[,p[,...]])` and `not(p)` where `p` is a predicate

Reviewer: Is "It suppose" proper grammar here?  I can't even tell.   "If suppose" maybe?

`field OP <value>` or `exists(field)`. `OP` is one of `=`, `!=`, `>`, `>=`, `<`, `<=`,
and `~`, `!~` (for regex matching).  If the value is a quoted string, comparisons are

> made a strings, where if the value is not quoted comparisons are attempted numerically.

Reviewer: ```suggestion
made as strings.  If the value is not quoted comparisons are attempted numerically.
```
